apps/api/src/common/env/env.schema.ts

```diff
@@ -75,7 +75,7 @@ const devSchema = z.object({
 
   SMTP_HOST: z.string(),
   SMTP_PORT: z.string(),
-  SMTP_SECURE: z.string(),
+  SMTP_SECURE: z.boolean(),
```

Comment:
You should be using `.transform()`. Refer to this: https://github.com/keyshade-xyz/keyshade/blob/0a0b80a2fcaacc3cbe19e0036103db79d751f8e8/apps/api/src/common/env/env.schema.ts#L53